### PR TITLE
HSD8-659 Adding bottom margin for group blocks to space them better

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Added bottom margin to group blocks

# Needed By (Date)
- no rush

# Urgency
- low

# Steps to Test
1. sync any site
1. create a basic page
1. edit the layout for that page
1. add 2 group blocks in a sidebar or someplace
1. view the page after saving and ensure the groups have space between them.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by sherakama** [comment]
You have a tonne of changes in this PR seemingly unrelated to the bottom margin. Are you sure you want all of this?

**Comment by pookmish** [comment]
wrong branch. sorry

**Comment by pookmish** [comment]
I updated branch to merge into. only 4 file changes

File: docroot/modules/humsci/hs_blocks/hs_blocks.libraries.yml [hunk 1]
```diff
@@ -3,3 +3,8 @@ login.admin:
   css:
     base:
       css/hs_blocks.login.admin.css: {}
+block_group:
```
Comment (sherakama) [inline docroot/modules/humsci/hs_blocks/hs_blocks.libraries.yml]: Everything else is called group_block, why change now?

**Review by sherakama — APPROVED** [review]

File: docroot/modules/humsci/hs_blocks/hs_blocks.libraries.yml (hunk 1 above)
Comment (pookmish) [reply docroot/modules/humsci/hs_blocks/hs_blocks.libraries.yml]: just checking if you're paying attention

**Comment by boznik** [comment]
can you stop the block from following the cursor and the css for clicking the link

**Review by boznik — APPROVED** [review]
This is working as expected.

**Pull request merged**